packages/components/storybook/spacing.stories.mdx

```diff
@@ -0,0 +1,13 @@
+import { Meta } from '@storybook/blocks'
+
+import { SpacingTable } from './SpacingTable'
+
+<Meta title="Design Tokens/Spacing" />
+
```

Comment:
Might be a special case of getting rid of the whitespace lines here? Looking at the image, it looks like there's actually whitespace above the Spacing heading which we probably don't want

Reply:
Or maybe we leave it, since the Colors token page also has it.

Reply:
Test this out and removing the whitespace has no effect on the whitespace above